src/app/[locale]/(app)/analytics/subscriptions/page.tsx

```
@@ -0,0 +1,645 @@
+              <option value='MX'>México</option>
+              <option value='PT'>Portugal</option>
+            </select>
+          </div>
+        </div>
+      </CardContent>
+    </Card>
+  )
+}
+
+// Subscription metrics component
+function SubscriptionMetrics({ 
```

ESLint

[error] Delete `·`

(prettier/prettier)